CONTRIBUTING.md

```diff
@@ -91,10 +91,23 @@ bash scripts/format_backend.sh
 bash scripts/format_frontend.sh
 ```
 
-You can run the circleci checks locally as well.
+## CI checks
+
+Currently the following checks are run in CI:
+
+```
```

Comment:
This code block here is for the client. Can we add a sub-header here for "python client" .

The other subheader can be "gradio library" and its contents should be

```
bash scripts/lint_backend.sh
bash scripts/type_check_backend.sh
python -m pytest -m "not flaky" --ignore=client
python -m pytest -m "flaky" --ignore=client
```

Do you mind making the change @2-5 ?

Reply:
Ok, I have made the changes.